.github/workflows/ci.yaml

```
@@ -35,7 +35,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.2, 7.3, 7.4, 8.0]
+                php: [7.2, 7.3, 7.4, 8.0, 8.1]
```

```suggestion
                php: ['7.2', '7.3', '7.4', '8.0', '8.1']
```

i recently realized that the versions should be string, not float. that will also make builds for 8.0 report as "8.0" instead of "8" ;-)